web/filament-js/utilities.js

```diff
@@ -59,6 +59,24 @@ Filament.PixelBuffer = function(typedarray, format, datatype) {
   return bd;
 };
 
+/// CompressedPixelBuffer ::function:: Constructs a [PixelBufferDescriptor] for compressed texture
+/// data by copying a typed array into the WASM heap.
+/// typedarray ::argument:: Data to consume (e.g. Uint8Array, Uint16Array, Float32Array)
+/// cdatatype ::argument:: [CompressedPixelDataType]
+/// ::retval:: [PixelBufferDescriptor]
+Filament.CompressedPixelBuffer = function(typedarray, cdatatype) {
+    console.assert(typedarray.buffer instanceof ArrayBuffer);
+    console.assert(typedarray.byteLength > 0);
+    if (Filament.HEAPU32.buffer == typedarray.buffer) {
+      typedarray = new Uint8Array(typedarray);
```

Comment:
The rest of the file uses 4 space indent

Reply:
Good catch, fixed.